src/session_stats.cpp

```diff
@@ -545,15 +548,14 @@ namespace libtorrent
 		METRIC(sock_bufs, socket_recv_size20)
 
 		// ... more
-	};
+	}});
 #undef METRIC
 
 	std::vector<stats_metric> session_stats_metrics()
 	{
-		std::vector<stats_metric> stats;
-		int const num = sizeof(metrics) / sizeof(metrics[0]);
-		stats.resize(num);
-		for (std::size_t i = 0; i < num; ++i)
+		aux::vector<stats_metric> stats;
```

Comment:
I don't know, how can I verify this? (since this is not the only place when `aux::*` is in the return for an `std::*` type)